main.cpp

```diff
@@ -13,75 +13,506 @@ What you create in this project will be used as the basis of Project 5 in the co
+        //the parameter name is related to the work the function will perform.
+        void fillTank(double fuelAmountInGallons = 2.0);  
+        void breakDown(std::string failureType, bool requiresTow = false);
+        int getMilesTraveledAnnually(bool includeUberLyftTrips);
+    };
+
+    //wash and wax car
+    void washAndWaxCar(Car carA); //4) a member function whose parameter is a UDT.
+    //charge customer
+    float chargeCustomer(float discountPercentage);
+    //detail the car interior
+    void detailInterior(Car carB);
+    
+    //5) a member variable whose type is a UDT.
+    Car carBeingServiced;  
+};
+
+/*
+Thing 1) Song
+5 properties:
+    1) duration of the song // double
+    2) tempo of the song // int 
+    3) key of the song // char
+    4) number of artists performing on the song // int
+    5) record label that released the song // string
+3 things it can do:
+    1) played on the radio
+    2) performed live at a concert
+    3) be remixed by another artist
+ */
+
+struct Song
+{
+    class Artist
+    {
+        std::string name = "Moby";
+        int Age = 50;
+        std::string birthPlace = "America";
+        int yearsActive = 25;
+        std::string mostPopularSong = "Porcelain";
+
+        void performSongLive(bool tour = true);
+        void recordSongInTheStudio(bool recordAlbum);
+        void promoteSong(int numberOfInterviews, bool tvAppearances);
+    };
+
+    Artist artistA;
+
+    // duration of the song
+    int duration = 5;
+    // tempo of the song
+    int tempo = 125;
+    // key of the song
+    char key = 'a';
+    // number of artists performing on the song
+    int numOfPerformingArtists = 4;
+    // record label that released the song
+    std::string recordLabel = "Virgin";
+    
+    // play song on the radio
+    void playSongOnRadio(int duration, int tempo);
+    // perform song live at a concert
+    void performSongLiveAtConcert(Artist artistB);
+    // remix the song
+    void remix(std::string title, Artist artistC, std::string recordLabel);
+};
+
+/*
+Thing 2) Sandwich
+5 properties:
+    1) type of bread // string
+    2) type of fillings // string
+    3) amount of different fillings // int
+    4) condiment used // string
+    5) number of calories // int
+3 things it can do:
+    1) cause foodpoisoning
+    2) provide a satisfying meal
+    3) go moldy if left uneaten
+ */
+
+struct Sandwich
+{
+    // type of bread
+    std::string bread = "brown";
+    // type of fillings
+    std::string fillings = "meat";
+    // amount of different fillings
+    int numOfFillings = 2;
+    // condiment used
+    std::string condiment = "mayo";
+    // number of calories
+    int calories = 600;
+
+    // cause foodpoisoning
+    void foodPoisoning(std::string fillings = "uncooked chicken");
+    // provide a satisfying meal
+    void satisfyingMeal(int calories);
+    // go moldy if left uneaten
+    void moldy(int daysWithoutStorage);
```

Comment:
none of these names contain the 'action' or verb.
change your function names to match the plain-english description
- [ ] revise